frontend/app/(tabs)/create.tsx

```diff
@@ -13,67 +15,185 @@ import {
   H5,
   Text,
   TextArea,
-  View,
+  Switch,
+  Form,
 } from 'tamagui';
-import type { TabsContentProps } from 'tamagui';
+import {
+  ByteSchema,
+  ByteSchemaInputs,
+  RecipeSchema,
+  RecipeSchemaInputs,
+} from '@/types/postInput';
+import { Controller, SubmitHandler, set, useForm } from 'react-hook-form';
+import UserInput from '@/components/UserInput';
 
 export default function Create() {
   const [isEnabled, setEnabledElements] = useState(false);
 
+  const {
+    handleSubmit: handleSubmitByte,
+    control: controlByte,
+    formState: { errors: errorsByte },
+  } = useForm<ByteSchemaInputs>({
+    resolver: zodResolver(ByteSchema),
+    defaultValues: {
+      description: '',
+    },
+  });
+
+  const {
+    handleSubmit: handleSubmitRecipe,
+    control: controlRecipe,
+    formState: { errors: errorsRecipe },
+  } = useForm<RecipeSchemaInputs>({
+    resolver: zodResolver(RecipeSchema),
+    defaultValues: {
+      descr: '',
+      ingredients: '',
+      steps: '',
+    },
+  });
+  const createByte: SubmitHandler<ByteSchemaInputs> = (data) => {
+    console.log('dfljkfd');
+    try {
+      console.log(data.description);
+    } catch (err) {
+      // error
+    }
+  };
+  const createRecipe: SubmitHandler<RecipeSchemaInputs> = (data) => {
+    try {
+      console.log(data.descr);
+      console.log(data.steps);
+      console.log(data.ingredients);
+    } catch (err) {
+      //error
+    }
+  };
+
   return (
     <View>
```

Comment:
You should use the `<SafeAreaView>` in case some of the stuff renders out of bounds of the screen.